master/internal/stream/system.go

```diff
@@ -35,6 +36,7 @@ const (
 type PublisherSet struct {
 	DBAddress string
 	Trials    *stream.Publisher[*TrialMsg]
+	Metrics   *stream.Publisher[*MetricMsg]
 	// Experiments *stream.Publisher[*ExperimentMsg]
```

Comment:
also thanks for stubbing this out a long time ago :)

Reply:
Haha I think that the thanks there is owed to @rb-determined-ai he added the stubs prior to us jumping in